### ISSUE TITLE
Feature Request: Manage model through UI instead of .env.local

### ISSUE THREAD

**Issue by TriEyeMonster** [huggingface/chat-ui#1286]
Since the website will be running in a production environment, it will be better that admin is able add or remove model through UI instead of a production deployment for a minor .env.local update.

**Comment by nsarrazin** [comment]
Yeah that would be definitely be nice, I think it was on our roadmap a while ago but never got prioritized. If there's strong demand we could try to tackle this! It would make life a lot easier for self-hosting users.

**Comment by TriEyeMonster** [comment]
Yeah, this is a essential feature if we want to make it more product ready.

**Comment by widyaageng** [comment]
+1 for prioritisation.

**Comment by neven4** [comment]
@nsarrazin how you want to handle `endpoints` array in the modal? Currently, we have a multiple types (I think it will be hard to unify them).

Do we want to have an additional mongo collection for each endpoint type (`AWS`, `openai`, `anthropic`), or just store types and json and validate it from the frontend side?
